### DISCUSSION TITLE
FreeRADIUS

### DISCUSSION THREAD

**Discussion by 68656c6c** [selfhst/icons#120] [New Icon Request]
Github project: https://github.com/FreeRADIUS/freeradius-server
Possible suggestion for icon: https://avatars.githubusercontent.com/u/2430370?s=200&v=4 (FreeRADIUS Github account image)
Maybe even ask FreeRADIUS what icon they would like to use?

**Comment by shollyethan** [comment]
FreeRADIUS has been added - thanks!
